image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml.htpl

```diff
@@ -84,6 +84,8 @@ spec:
         - name: ROX_TELEMETRY_STORAGE_KEY_V1
           value: {{ ._rox.central.telemetry.storage.key | quote }}
         {{- end }}
+        - name: ROX_TELEMETRY_API_WHITELIST
```

Comment:
How about calling it 
```suggestion
        - name: ROX_TELEMETRY_API_ALLOWLIST
```

Reply:
Unfortunately, I'm not introducing a new environment variable, but rather [setting an existing one](https://github.com/stackrox/stackrox/blame/7dab2df3acfb46f997c06b312cd599ad6b6b1425/central/telemetry/centralclient/instance_config.go#L27). I'd be in favor of calling it `ALLOWLIST` as well, but I don't think it would be wise to rename it.